discovery_engine_flutter/example/.gitignore

```diff
@@ -44,3 +44,14 @@ app.*.map.json
 /android/app/debug
 /android/app/profile
 /android/app/release
+
+# Assets related
+/assets/*
+/assets/kpe_v0001/*
+/assets/smbert_v0000/*
```

Comment:
isn't this ignored anyways due to the glob ignore in l.49?

Reply:
it wasn't ignored, but I've changed it to a onliner now